NAMESPACE

```diff
@@ -54,6 +54,7 @@ export(logToLin)
 export(metadata)
 export(mixedToLog)
 export(observeSolute)
+export(plotEGRET)
```

Comment:
I'm interested in keeping it internal until it's more generic. not against exporting it in future, but trying to stick to MVP for Rich and Ana's needs for the month.